templates/home/api-en.html

```diff
@@ -22,19 +22,24 @@
 	<dd>Template <!-- Common design patter --></dd>
 
 	<dt>Last review</dt>
-	<dd>2022-04-27</dd>
+	<dd>2022-06-29</dd>
 
 	<dt>Conforming to</dt>
 	<dd>Going to be: Content and IA spec 2.0</dd>
 
 	<dt>Specification</dt>
 	<dd><a href="https://design.canada.ca/mandatory-templates/home-page.html">Home page for Canada.ca</a>
+
+	<dt>Availble instances</dt>
+	<dd>https://www.canada.ca/en.html</dd>
```

Comment:
Should be links <a> and in the same <dd> element (could be separated by a comma "," or the word "and").